tests/test_doublyrobust.py

```diff
@@ -0,0 +1,184 @@
+        tmle.fit()
+        npt.assert_allclose(tmle.psi, r_rr)
+
+    def test_match_r_tmle_rr_ci(self, df):
+        r_ci = 0.2773936, 1.0296262
+        tmle = TMLE(df, exposure='art', outcome='dead', psi='risk_ratio')
+        tmle.exposure_model('male + age0 + age_rs1 + age_rs2 + cd40 + cd4_rs1 + cd4_rs2 + dvl0', print_results=False)
+        tmle.outcome_model('art + male + age0 + age_rs1 + age_rs2 + cd40 + cd4_rs1 + cd4_rs2 + dvl0',
+                           print_results=False)
+        tmle.fit()
+        npt.assert_allclose(tmle.confint, r_ci, rtol=1e-5)
+
+    def test_match_r_tmle_oddsratio(self, df):
+        r_rr = 0.4844782
+        tmle = TMLE(df, exposure='art', outcome='dead', psi='odds_ratio')
+        tmle.exposure_model('male + age0 + age_rs1 + age_rs2 + cd40 + cd4_rs1 + cd4_rs2 + dvl0', print_results=False)
+        tmle.outcome_model('art + male + age0 + age_rs1 + age_rs2 + cd40 + cd4_rs1 + cd4_rs2 + dvl0',
+                           print_results=False)
+        tmle.fit()
+        npt.assert_allclose(tmle.psi, r_rr)
+
+    def test_match_r_tmle_or_ci(self, df):
+        r_ci = 0.232966, 1.007525
+        tmle = TMLE(df, exposure='art', outcome='dead', psi='odds_ratio')
+        tmle.exposure_model('male + age0 + age_rs1 + age_rs2 + cd40 + cd4_rs1 + cd4_rs2 + dvl0', print_results=False)
+        tmle.outcome_model('art + male + age0 + age_rs1 + age_rs2 + cd40 + cd4_rs1 + cd4_rs2 + dvl0',
+                           print_results=False)
+        tmle.fit()
+        npt.assert_allclose(tmle.confint, r_ci, rtol=1e-5)
+
+    def test_symmetric_bounds_on_gW(self, df):
+        r_rd = -0.08203143
+        r_ci = -0.1498092, -0.01425363
+        tmle = TMLE(df, exposure='art', outcome='dead')
+        tmle.exposure_model('male + age0 + age_rs1 + age_rs2 + cd40 + cd4_rs1 + cd4_rs2 + dvl0',
+                            bound=0.1, print_results=False)
+        tmle.outcome_model('art + male + age0 + age_rs1 + age_rs2 + cd40 + cd4_rs1 + cd4_rs2 + dvl0',
+                           print_results=False)
+        tmle.fit()
+        npt.assert_allclose(tmle.psi, r_rd)
+        npt.assert_allclose(tmle.confint, r_ci, rtol=1e-5)
+
+    def test_asymmetric_bounds_on_gW(self, df):
+        r_rd = -0.08433208
+        r_ci = -0.1541296, -0.01453453
+        tmle = TMLE(df, exposure='art', outcome='dead')
+        tmle.exposure_model('male + age0 + age_rs1 + age_rs2 + cd40 + cd4_rs1 + cd4_rs2 + dvl0',
+                            bound=[0.025, 0.9], print_results=False)
+        tmle.outcome_model('art + male + age0 + age_rs1 + age_rs2 + cd40 + cd4_rs1 + cd4_rs2 + dvl0',
+                           print_results=False)
+        tmle.fit()
+        npt.assert_allclose(tmle.psi, r_rd)
+        npt.assert_allclose(tmle.confint, r_ci, rtol=1e-5)
+
+    def test_sklearn_in_tmle(self, df):
+        log = LogisticRegression(penalty='l1', random_state=201)
+        tmle = TMLE(df, exposure='art', outcome='dead', psi='risk_difference')
```

Comment:
Hmmm that's good. I originally chose ``psi`` since that is what van der Laan defines the target parameter as. It can be defined any expected value of interest (like risk difference, risk ratio, risk, etc.). I will think about this